[html] Restore coverage for query encoding

jugglinmike opened the pull request:
When the query encoding navigation tests were refactored to use WPT's
"variant" feature [1], the new structure omitted one of the encodings
which was previously tested. Although the encoding will be used when a
variant is not specified, the automation tooling only uses variants
which are explicitly specified. This can be observed through the
`--list-tests` flag of the `wpt run` command:

    $ ./wpt run --list-tests firefox html/infrastructure/urls/resolving-urls/query-encoding/navigation.sub.html
    /html/infrastructure/urls/resolving-urls/query-encoding/navigation.sub.html?encoding=utf8
    /html/infrastructure/urls/resolving-urls/query-encoding/navigation.sub.html?encoding=x-cp1251

Explicitly declare the default variant, restoring the original test
coverage.

[1] https://github.com/web-platform-tests/wpt/pull/11300

jugglinmike committed:
[html] Restore coverage for query encoding

When the query encoding navigation tests were refactored to use WPT's
"variant" feature [1], the new structure omitted one of the encodings
which was previously tested. Although the encoding will be used when a
variant is not specified, the automation tooling only uses variants
which are explicitly specified. This can be observed through the
`--list-tests` flag of the `wpt run` command:

    $ ./wpt run --list-tests firefox html/infrastructure/urls/resolving-urls/query-encoding/navigation.sub.html
    /html/infrastructure/urls/resolving-urls/query-encoding/navigation.sub.html?encoding=utf8
    /html/infrastructure/urls/resolving-urls/query-encoding/navigation.sub.html?encoding=x-cp1251

Explicitly declare the default variant, restoring the original test
coverage.

[1] https://github.com/web-platform-tests/wpt/pull/11300

annevk commented:
Could you also update the other .sub.html files in that directory? Thanks for catching this!

jugglinmike committed:
[html] Improve coverage for query encoding

Although the `windows-1252` character set will be used when a variant is
not specified, the automation tooling only uses variants which are
explicitly specified. Explicitly declare the default variant in order to
extend test coverage.

jugglinmike commented:
You got it, @annevk

annevk commented:
Thanks again!
